plasmapy/formulary/tests/test_quantum.py

```diff
@@ -156,15 +156,25 @@ def setup_class(self):
         self.n_e = 1e20 * u.cm**-3
         self.n_e_fail = 1e23 * u.cm**-3
         self.T = 11604 * u.K
-        self.True1 = 6.133671348607095e-11
+        self.True1 = 0
 
-    def test_known1(self):
+    @staticmethod
+    @pytest.mark.parametrize(
+        "n_e, T, expected_beta",
+        [
+            (1e20 * u.cm**-3, 11604 * u.K, 0),
+            (1e25 * u.cm**-3, 11604 * u.K, 268.68166791746324),
+        ],
+    )
+    def test_knowns(n_e, T, expected_beta):
         """
-        Tests Fermi_integral for expected value.
+        Tests chemical_potential for expected value.
         """
-        methodVal = chemical_potential(self.n_e, self.T)
-        testTrue = u.isclose(methodVal, self.True1, rtol=1e-16, atol=0.0)
-        errStr = f"Chemical potential value should be {self.True1} and not {methodVal}."
+        methodVal = chemical_potential(n_e, T).value
+        testTrue = np.isclose(methodVal, expected_beta, rtol=1e-16, atol=0.0)
+        errStr = (
+            f"Chemical potential value should be {expected_beta} and not {methodVal}."
+        )
         assert testTrue, errStr
```

Comment:
Can you add another `assert` to ensure the returned unit is `dimensionless_unscaled`?  It'll likely look something like...

```python
assert testTrue, errStr
assert chemical_potential.unit == u.dimensionless_unscaled
```

Also, could you please refactor this to meet PEP8 naming standards?  e.g. `methodVal` -> `method_val`